src/app/amsterdam/contact/vraag/page.tsx

```diff
@@ -20,32 +31,30 @@ function Question() {
       <Grid.Cell span={{ narrow: 4, medium: 6, wide: 8 }} start={{ narrow: 1, medium: 2, wide: 3 }}>
         <Breadcrumb>
           <NextLink legacyBehavior passHref href="/amsterdam">
-            <Breadcrumb.Item href="/amsterdam">Home</Breadcrumb.Item>
+            <Breadcrumb.Link href="/amsterdam">Home</Breadcrumb.Link>
           </NextLink>
         </Breadcrumb>
         <form className="ams-gap--md" onSubmit={handleSubmit}>
           <Heading>Contact</Heading>
-          <div>
+          <Field>
             <Label htmlFor="description">Wat wilt u aan de gemeente vragen?</Label>
             <Paragraph size="small">
               Een duidelijke beschrijving van uw vraag helpt ons bij het behandelen. U kunt eventueel nog een bijlage
               toevoegen.
             </Paragraph>
             <TextArea id="description" rows={4} />
             <Paragraph size="small">0/1000 tekens</Paragraph>
-          </div>
-          <Fieldset legend="U kunt hier een bijlage toevoegen">
-            <input
-              type="file"
-              id="fileUpload"
-              data-testid="file-input-upload"
+          </Field>
+          <FieldSet legend="U kunt hier een bijlage toevoegen">
+            <FileInput
               accept="image/jpeg,image/jpg,image/png,image/gif"
-              name="bijlage"
-              multiple
               aria-label="Selecteer een bestand"
```

Comment:
Shouldn't this just have a Label component connected to it, instead of an `aria-label`?

Reply:
A `Label` has already been introduced by replacing the `FieldSet` with a `Field`.

The label is ‘U kunt hier een bijlage toevoegen’. The control reads ‘Selecteer een bestand’ as an instruction. Feels right, but it is not correct?

Reply:
NVDA now only reads 'Selecteer een bestand. Geen bestand gekozen.'. 'U kunt hier een bijlage toevoegen' is overruled by the `aria-label`. I don't think we want that?

Reply:
We now have this. VoiceOver on iOS reads it out correctly (except for mixing English and Dutch). What would you propose?

```jsx
<Field>
  <Label htmlFor="fileUpload">U kunt hier een bijlage toevoegen</Label>
  <FileInput
    accept="image/jpeg,image/jpg,image/png,image/gif"
    aria-label="Selecteer een bestand"
    data-testid="file-input-upload"
    id="fileUpload"
    multiple
    name="bijlage"
  />
</Field>
```

Reply:
Does VoiceOver read both 'U kunt hier een bijlage toevoegen' and 'Selecteer een bestand'? 

I'd opt for removing the `aria-label`, so the visible text is the same as the read text. So something like this:

```
<Field>
  <Label htmlFor="fileUpload">U kunt hier een bijlage toevoegen</Label>
  <FileInput
    accept="image/jpeg,image/jpg,image/png,image/gif"
    data-testid="file-input-upload"
    id="fileUpload"
    multiple
    name="bijlage"
  />
</Field>
```

Reply:
Done this for now and removed the test ids as well.